geode-for-redis/src/test/java/org/apache/geode/redis/internal/data/RedisListTest.java

```diff
@@ -110,6 +118,58 @@ public void equals_returnsTrue_givenDifferentEmptyLists() {
     assertThat(list2).isEqualTo(list1);
   }
 
+  @Test
+  public void lrem_storesStableDelta_inOrderRemove() {
+    Region<RedisKey, RedisData> region = uncheckedCast(mock(PartitionedRegion.class));
+    when(region.put(any(), any())).thenAnswer(this::validateDeltaSerialization);
+
+    byte[] element = new byte[] {1};
+    RedisList list = createRedisListWithDuplicateElements();
+
+    list.lrem(2, element, region, null);
+
+    verify(region).put(any(), any());
+    assertThat(list.hasDelta()).isFalse();
+  }
+
+  @Test
+  public void lrem_storesStableDelta_reverseOrderRemove() {
```

Comment:
Since in-order removing and reverse-order removing now both use the same Delta type, this test is redundant.